services/libfuzzer/setup.sh

```diff
@@ -39,6 +39,7 @@ packages=(
   less
   libglu1-mesa
   libosmesa6
+  libpci-dev
```

Comment:
This should be `libpci3` instead. `libpci-dev` will pull in headers which aren't necessary.